packages/grafana-ui/src/components/Combobox/MultiCombobox.tsx

```diff
@@ -158,54 +174,72 @@ export const MultiCombobox = <T extends string | number>(props: MultiComboboxPro
       }
     },
 
+    onIsOpenChange: ({ isOpen, inputValue }) => {
+      if (isOpen && inputValue === '') {
+        updateOptions(inputValue);
+      }
+    },
+
     onStateChange: ({ inputValue: newInputValue, type, selectedItem: newSelectedItem }) => {
       switch (type) {
         case useCombobox.stateChangeTypes.InputKeyDownEnter:
         case useCombobox.stateChangeTypes.ItemClick:
+          // TODO: i don't like multiple breaks
+          if (!newSelectedItem) {
+            break;
```

Comment:
Feel like maybe `return` would be better here? Otherwise you have to check it the `switch` is the only thing in the function.